src/server/warnings/index.js

```diff
@@ -0,0 +1,9 @@
+export default function (kbnServer, server, config) {
+  if (kbnServer.config.get('env.prod')) {
+    return;
+  }
+
+  process.on('warning', (warning) => {
+    server.log(['warning'], warning);
```

Comment:
We should add a `'process'` tag so that these warnings can be identified specifically